apps/admin/providers/accessCheck.ts

```diff
@@ -0,0 +1,46 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import {
+  fetchActionToAction,
+  ReactAdminFetchActions,
+  Permissions,
+  canAccess,
+} from "ra-data-simple-prisma";
+import { authProvider } from "./authProvider";
+import { authOptions } from "../pages/api/auth/[...nextauth]";
+import { getServerSession } from "next-auth";
+
+export const extractAccessObjFromReq = async (
```

Comment:
this is the only file that I couldn't abstract to be used in the package

Reply:
right, it's too dependent on nextjs